generic-database-plugin/widgets/Database-batchsink.json

```diff
@@ -17,20 +17,12 @@
         },
         {
           "widget-type": "textbox",
-          "label": "Plugin Name",
-          "name": "jdbcPluginName"
+          "label": "Driver Name",
+          "name": "jdbcDriverName"
         },
         {
           "widget-type": "textbox",
-          "label": "Plugin Type",
-          "name": "jdbcPluginType",
-          "widget-attributes": {
-            "default": "jdbc"
-          }
-        },
-        {
-          "widget-type": "textbox",
-          "label": "Connection String",
+          "label": "Connection string",
```

Comment:
By convention, labels capitalize most words. Let's keep it 'Connection String' to make it consistent with everything else.

Reply:
fixed